FrontEnd/src/components/AboutPage/MeetTheTeam/MeetTheTeam.tsx

```diff
@@ -1,7 +1,126 @@
+      'https://drive.google.com/uc?export=view&id=1Gyusdebr_JVrgGYllwV5ODkNIAZot6ca',
+    title: 'Developer',
+
+    quote: { number: 1, length: 'month' },
+    linkedIn: 'https://www.linkedin.com/in/willhsu25/',
+  },
+  {
+    name: 'Bao Hoang',
+    imgSrc:
+      'https://drive.google.com/uc?export=view&id=1qeWNQr7q6tr5-LAbXDj119X2IEAtfgDr',
+    title: 'Developer',
+
+    quote: { number: 1, length: 'month' },
+    linkedIn: 'https://www.linkedin.com/in/bao-hoang-g/',
+  },
+  {
+    name: 'Anya Chandorkar',
+    imgSrc:
+      'https://drive.google.com/uc?export=view&id=1OO5izg1pYR0Fto9KIPyyyw1Ztd1rfgbZ',
+    title: 'Developer',
+
+    quote: 'Always open to learning new things',
+    linkedIn: 'https://www.linkedin.com/in/anya-chandorkar',
+  },
+  {
+    name: 'Cris Zong',
+    imgSrc:
+      'https://drive.google.com/uc?export=view&id=11no-IR8qfhckkuljWFhuQQELwgqCmrWN',
+    title: 'Business',
+
+    quote: 'Life is Hard',
+    linkedIn: 'https://www.linkedin.com/in/criszong/',
+  },
+  {
+    name: 'Amit Bar',
+    imgSrc:
+      'https://drive.google.com/uc?export=view&id=13UvvbjYgK2CD_I1YX-dVwDizhaQTfs9i',
+    title: 'Developer',
+
+    quote: { number: 1, length: 'year' },
+    linkedIn: 'https://www.linkedin.com/in/amitbar/',
+  },
+  {
+    name: 'Mohit Gayatri',
+    imgSrc:
+      'https://drive.google.com/uc?export=view&id=1B5zTq802y08q1-L-95hEVAIP-dYAcA-r',
+    title: 'Developer',
+
+    quote: 'Backend software engineer at Homehub',
+    linkedIn: 'https://www.linkedin.com/in/mohit-gayatri-597180127/',
+  },
+  {
+    name: 'Rahul Datta',
+    imgSrc:
+      'https://drive.google.com/uc?export=view&id=1ouywZKH9HCtd-Yet8wuWvoPtHFT30vql',
+    title: 'Business',
+
+    quote: { number: 1, length: 'year' },
+    linkedIn: 'https://www.linkedin.com/in/rahuldatta10',
+  },
+  {
+    name: 'Saarthak Trivedi',
+    imgSrc:
+      'https://drive.google.com/uc?export=view&id=1hRHSoFD7HdCgOkmvueJySEp_EkURENFG',
+    title: 'Developer',
+
+    quote: { number: 1, length: 'year' },
+    linkedIn: 'https://www.linkedin.com/in/saarthak-trivedi-560abb185/',
+  },
+  {
+    name: 'Elias Rose',
+    imgSrc:
+      'https://drive.google.com/uc?export=view&id=1ySpLBOZWfLY6IogYceLnazvcIaWhSufu',
+    title: 'Designer',
+
+    quote: 'Home is where the desktop is.',
+    linkedIn: 'https://www.linkedin.com/in/elias-rose-33136a15a',
+  },
+  {
+    name: 'Ali Long',
+    imgSrc:
+      'https://drive.google.com/uc?export=view&id=1PbxKW1VU975Sa5uA_4wiIJDtz90fV4OP',
+    title: 'Designer',
+    quote: 'Sleep early, sleep often',
+    website: 'http://yingxilong.com/',
+    linkedIn: 'https://www.linkedin.com/in/yingxi-long/',
+  },
+  {
+    name: 'Anthony Manrique',
+    imgSrc:
+      'https://drive.google.com/uc?export=view&id=1KuA4pJfodmaJWLK8bjXoE4sTqccfCOtu',
+    title: 'Developer',
+    quote: 'True',
+    linkedIn: 'http://linkedin.com/in/anthony-manrique',
+  },
+];
+
+const MeetTheTeam: FunctionComponent = () => {
+  return (
```

Comment:
In javascript, the following are the same:
() => "this is a returned string";
() => {
    return "this is a returned string";
}

If you exclude the curly braces, the next statement will be returned. You can use that in this function
const MeetTheTeam: FunctionComponent = () => (
    <div .....
    ....
);